src/components/Markdown/Markdown.scss

```diff
@@ -19,14 +19,14 @@
     margin:1.5em 0 0.25em;
     color:getColor(fiord);
 
-    &:first-child {
-      margin-top: 0;
-      line-height: 1;
-    }
-
     tt, code { font-size: 90%; color: inherit }
   }
 
+  h1:first-child {
```

Comment:
What is the reason of this change?

Reply:
This is because, when a README has an **h2** heading as the first child, if we remove the margin it gets collapsed with the page title and it looks bad.